core/vtk/ttkAlgorithm/ttkUtils.cpp

```diff
@@ -359,3 +362,26 @@ int ttkUtils::CellVertexFromPoints(vtkDataSet *const dataSet,
 
   return 1;
 }
+
```

Comment:
it seems that the functions `startMPITimer` and `endMPITimer` can go to `core/base/common/MPIUtils.h`.
everything that does not include VTK things can be moved down there.
(this was implied in our previous discussion with @m-s-will on the topic)